src/content/docs/infrastructure/prometheus-integrations/integrations-list/calico-integration.mdx

```diff
@@ -0,0 +1,87 @@
+
+* Network connectivity between workloads.
+* Network security policy enforcement between workloads.
+
+Use New Relic to view a dashboard based on Prometheus metrics which helps you understand the Calico CNI of your k8s cluster. With New Relic you can monitor:
+
+* Active network policies by instances
+* Active IP tables rules by instances and chain type (filter, mangle, nat, and raw)
+* IP tables save and restore errors
+* BPF specific metrics if BPF is used as a dataplane for Calico
+
+<img
+  title="Calico Dashboard"
+  alt="Calico Dashboard"
+  src={kubernetesCalicoDashboard}
+/>
+
+## Enable the integration
+
+Follow these steps to enable the integration.
+
+1. Follow the [Calico documentation](https://projectcalico.docs.tigera.io/maintenance/monitor/monitor-component-metrics) for Prometheus to discover the Calico metrics endpoints.
+
+2. Set up Prometheus monitoring. Prometheus metrics needs to be integrated with New Relic, you can use the Prometheus Agent or the Remote Write integration, see [how to send Prometheus metrics](/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/).
+
+3. Use the following query to confirm metrics are being ingested as expected:
+
+    ```
+    FROM Metric SELECT count(*) WHERE metricName LIKE 'felix_%' FACET metricName LIMIT MAX
+    ```
+
+4. Install the [Calico quickstart](https://newrelic.com/instant-observability/calico-newrelic) to access built-in dashboards and and [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/).
+
+  Once you imported, you can edit or clone the assets to adapt them to your specific requirements. 
+
+## Find and use the data
+
+### Metrics
+
+Prometheus metrics are stored as dimensional metrics. You can [query using NRQL](/docs/telemetry-data-platform/get-data/apis/query-metric-data-type/) or use the [Data Explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/) to browse the available metrics, facet, and filter by the associated dimensions.
+
+The different sets of metrics exposed by this integration are defined in the [Calico documentation](https://projectcalico.docs.tigera.io/reference/felix/prometheus).
+
+Use the following NRQL queries to understand the metrics being ingested in New Relic:
+
+* List unique metric names:
+
+  ```
+  FROM Metric SELECT uniques(metricName) WHERE metricName LIKE 'felix_%' LIMIT MAX
+  ```
+
+* Count number of metric updates:
+
+  ```
+  FROM Metric SELECT datapointcount() WHERE metricName LIKE 'felix_%' LIMIT MAX
+  ```
+
+* Estimate data ingestion (daily ingest, in bytes):
+
+	```
+  FROM Metric SELECT bytecountestimate() WHERE metricName LIKE 'felix_%' SINCE
+1 day ago
+  ```
+
+## Troubleshooting
+
+* Use this command to verify that he Calico Prometheus endpoint is emitting metrics on any K8s node configured with Calico CNI:
```

Comment:
I am writing merely here, but all integrations are currently missing any info regarding `app.kubernetes/name`

Reply:
Let me know what info I should add, please. Thanks!!

Reply:
@mangulonr will you address this?

Reply:
Done